metrics-operator/go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.10.1 // indirect
```

Comment:
Please remove all changes in this file, there should be no dependency bumps in this PR, same for go.sum

Reply:
Alright